frequency-chain/frequency-xyz
# 53 users section

# Description
- Users section placeholder image, content and styling in all screen sizes
- Closes #53 

## Screenshot(s)
<img width="873" alt="Screenshot 2024-11-13 at 4 40 11 PM" src="https://github.com/user-attachments/assets/e51f4837-c7c7-40bb-84c1-d522577a720a">
<img width="503" alt="Screenshot 2024-11-13 at 4 40 01 PM" src="https://github.com/user-attachments/assets/6190d0fe-49af-461c-9d9e-f1afee89d29d">
<img width="355" alt="Screenshot 2024-11-13 at 4 39 49 PM" src="https://github.com/user-attachments/assets/03a0512e-5989-4151-b3bb-e4dc3f6aa2ba">


## Type of change


# How to Test?

1. Pull and run locally
2. Go to UI
3. See that it looks like mocks in all screen sizes

# Checklist:
